tests/test_mdm.py

```diff
@@ -0,0 +1,69 @@
+"""Methods for testing the MDM classifier module."""
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.spd_matrices import SPDMetricAffine
+from geomstats.learning.mdm import RiemannianMinimumDistanceToMeanClassifier
+
+
+e = gs.exp(1.)
+
+
+class TestRiemannianMinimumDistanceToMeanClassifier(geomstats.tests.TestCase):
+    """Test of Riemannian MDM classifier."""
+
+    @geomstats.tests.np_only
+    def test_fit(self):
+        """Test the fit method."""
+        n_clusters = 2
+        MDMEstimator = RiemannianMinimumDistanceToMeanClassifier(
+            SPDMetricAffine(n=2), n_clusters, point_type='matrix')
+
+        points_A = gs.array([[[e ** 2, 0], [0, 1]],
```

Comment:
No upper case in variable names in python:
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Refactored.